dist/index.js

```diff
@@ -3339,6 +3607,9 @@ const PipelinesApiAxiosParamCreator = function (configuration) {
             // authentication BearerAuth required
             // http bearer authentication required
             yield (0, common_1.setBearerAuthToObject)(localVarHeaderParameter, configuration);
+            if (xExpireAfter != null) {
+                localVarHeaderParameter['x-expire-after'] = String(xExpireAfter);
+            }
```

Comment:
Here we are generating 'x-expire-after' but in other parts of the *.ts code like e.g. https://github.com/vmware-labs/vmware-image-builder-action/pull/156/files#diff-d9e1299192818bc4ed9ab5c7a5a2fbba102848f023caefd2216a7c0e7629c547L69 we are using "X-Expires-After". Is that correct?